interactions.go

```diff
@@ -326,8 +326,9 @@ const (
 
 // InteractionResponse represents a response for an interaction event.
 type InteractionResponse struct {
-	Type InteractionResponseType  `json:"type,omitempty"`
-	Data *InteractionResponseData `json:"data,omitempty"`
+	Type  InteractionResponseType  `json:"type,omitempty"`
+	Files []*File                  `json:"-"`
```

Comment:
It seems to me like this field would be better situated inside `InteractionResponseData` next to `Content`, `Embeds`, etc. just like `WebhookEdit`.

Reply:
My original reasoning was that Files aren't really part of the response structure, more just  things that get passed alongside the response. You could even return  just `{"type":4}` as the response if it contains files as well.